docs/design/human-in-the-loop/human-in-the-loop.md

```
@@ -0,0 +1,655 @@
+
+        user_input = input("Approve? (y/n): ")
+
+        # 3. Submit approval
+        requests.post(f"/approvals/{approval['id']}", json={
+            "approve": user_input.lower() == "y",
+            "reason": "User decision"
+        })
+
+    # 4. Continue conversation (or re-query)
+    # The next query in the same conversation will continue from where it left off
+```
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)